web/common.h

```diff
@@ -0,0 +1 @@
+#define MSG_NOSIGNAL 0 
```

Comment:
Alright, perhaps it is because I'm compiling with `clang` instead of `gcc` in my host system? I'm getting the following error if I don't include the `common.h`:

```
mba-3:web romanvg$ make
../esp82xx/common.mf:30: Warning: No shell variable 'ESP_ROOT', using '/Users/romanvg/esp8266/esp-open-sdk'
gcc -DICACHE_FLASH -DWEB_PORT=80 -DCOM_PORT=7777 -DBACKEND_PORT=7878  execute_reflash.c md5.c -o execute_reflash  -lusb-1.0
execute_reflash.c:65:39: error: use of undeclared identifier 'MSG_NOSIGNAL'
                return sendto( sockfd, buffer, len, MSG_NOSIGNAL, (struct sockaddr *)&servaddr,sizeof(servaddr));
                                                    ^
```

And a bunch of warnings too:

```
execute_reflash.c:85:5: warning: passing 'char [10000]' to parameter of type 'unsigned char *' converts between pointers to integer types with different sign [-Wpointer-sign]
                                recvline, //wLength  (more like max length)
                                ^~~~~~~~
/usr/local/include/libusb-1.0/libusb.h:1742:17: note: passing argument to parameter 'data' here
        unsigned char *data, uint16_t wLength, unsigned int timeout);
                       ^
execute_reflash.c:180:16: warning: passing 'char [64]' to parameter of type 'uint8_t *' (aka 'unsigned char *') converts between pointers to integer types with different sign
      [-Wpointer-sign]
                                SendData(  se, sel );
                                           ^~
execute_reflash.c:38:25: note: passing argument to parameter 'buffer' here
int SendData( uint8_t * buffer, int len )
                        ^
execute_reflash.c:207:14: warning: passing 'char [2000]' to parameter of type 'uint8_t *' (aka 'unsigned char *') converts between pointers to integer types with different sign
      [-Wpointer-sign]
                        SendData( bufferout, sendsize + r );
                                  ^~~~~~~~~
execute_reflash.c:38:25: note: passing argument to parameter 'buffer' here
int SendData( uint8_t * buffer, int len )
                        ^
execute_reflash.c:376:12: warning: passing 'char [1024]' to parameter of type 'uint8_t *' (aka 'unsigned char *') converts between pointers to integer types with different sign
      [-Wpointer-sign]
        SendData( cmd, strlen(cmd) );
                  ^~~
execute_reflash.c:38:25: note: passing argument to parameter 'buffer' here
int SendData( uint8_t * buffer, int len )
                        ^
execute_reflash.c:378:12: warning: passing 'char [1024]' to parameter of type 'uint8_t *' (aka 'unsigned char *') converts between pointers to integer types with different sign
      [-Wpointer-sign]
        SendData( cmd, strlen(cmd) );
                  ^~~
execute_reflash.c:38:25: note: passing argument to parameter 'buffer' here
int SendData( uint8_t * buffer, int len )
                        ^
5 warnings and 1 error generated.
make: *** [execute_reflash] Error 1
```